dt/dcn/edpm-post-ceph/nodeset/kustomization.yaml

```diff
@@ -148,12 +148,12 @@ replacements:
   - source:
       kind: ConfigMap
       name: service-values
-      fieldPath: data.cinderAPI.customServiceConfig
+      fieldPath: data.cinder.customServiceConfig
     targets:
       - select:
           kind: OpenStackControlPlane
         fieldPaths:
-          - spec.cinder.template.cinderAPI.customServiceConfig
+          - spec.cinder.template.cinder.customServiceConfig
```

Comment:
I believe this should be:
 spec.cinder.template.cinder.customServiceConfig
So the generated CR would be:
```
  cinder:
    template:
      customServiceConfig: |
        [DEFAULT]
        storage_availability_zone = az0
```
And not
```
  cinder:
    template:
      cinder:
        customServiceConfig: |
          [DEFAULT]
          storage_availability_zone = az
```